Src/FluentAssertions/CallerIdentifier.cs

```diff
@@ -267,4 +261,18 @@ private static bool IsBooleanLiteral(string candidate)
     {
         return candidate is "true" or "false";
     }
+
+    private static StackFrame[] GetFrames(StackTrace stack)
+    {
+        var frames = stack.GetFrames();
+#if !NET6_0_OR_GREATER
```

Comment:
I had to look up why this might be necessary for .NET Framework.

[net48](https://referencesource.microsoft.com/#mscorlib/system/diagnostics/stacktrace.cs,601)
[net core 3.0+](https://source.dot.net/#System.Private.CoreLib/src/libraries/System.Private.CoreLib/src/System/Diagnostics/StackTrace.cs,165)
[change](https://github.com/dotnet/coreclr/pull/25303/files#diff-cf1bd43d732ab10d08dc2126f73384119656794951312c072eff4b21c34c3bebL165-R165)

Reply:
Why not just `#if NET47`? I find this easier to understand 🤷‍♂️

Reply:
Because `NETSTANDARD` also is affected...

Reply:
Oh.. I see :)